gpic/gpic.go

```diff
@@ -115,10 +118,46 @@ func NewAvatar(inputs ...string) (Avatar, error) {
 		}
 
 		// check for GitHub ID
-		id, found := strings.CutPrefix(input, "ghi:")
+		gh_id, found := strings.CutPrefix(input, "ghi:")
```

Comment:
These variable names don't need to be changed. Doing so creates additional variables, using more memory, and isn't actually needed.

Reply:
yes